uiowa/uiowa
# phpcs - some fixes

- Removed unused var
- adjusted use of var within `t()`
- changed `access administration pages` permissions check to `administer basic site settings` which only webmaster and above roles have
- keep it under 80

# To Test

- Checkout `main` and pull
- Run `ddev blt validate:phpcs` - see warnings.
- Checkout `random-phpcs-fixes`
- Run `ddev blt validate:phpcs` - see solutions.
- Sync a site and go to Region Settings and see that the h3s are still rendering as part of each fieldset.
